src/modules/General/components/Avatar/avatar.types.ts

```diff
@@ -0,0 +1,15 @@
+import { CSSProperties } from 'react';
```

Comment:
if you can, please change to `index.types.ts` for all components you bring, but if you can not, let's skip it and do this convention for our new components